src/java.base/share/classes/jdk/internal/misc/CDS.java

```diff
@@ -59,4 +68,20 @@
      * Check if sharing is enabled via the UseSharedSpaces flag.
      */
     public static native boolean isSharingEnabled();
+
+    /**
+     * check if -XX:+DumpLoadedClassList and given file is open
+     */
+    public static boolean isDumpLoadedClassList() {
+        return isDumpLoadedClassList;
+    }
+
+    /**
+     * log output to DumpLoadedClassList
+     */
+    public static void logTraceResolve(String line) {
```

Comment:
`logTraceResolve` is too generic. How about `CDS.logLambdaFormInvoker()` to match the `@lambda-form-invoker` in the classlist file?